### ISSUE TITLE
拜拜苹果，那怕说的是2022年的December

### ISSUE THREAD

**Issue by liyanggyang** [kingToolbox/WindTerm#429]

**Comment by Asura88** [comment]
拜拜苹果，那怕说的是2022年的December #429

**Comment by kingToolbox** [comment]
Sorry, I know it is very disappointing that the MacOS version was not released at the expected time. But believe me, although there are many unexpected things, I am still working hard to develop and test the MacOS version. Like you, I hope to release a new stable version as soon as possible.

At present, two important bugs have been fixed, which has greatly improved the stability, and testing is continuing. Once the testing is completed, it will be released. Please be patient, thank you.

**Comment by Asura88** [comment]
兄弟，看你这英文，像是中国人哦



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年1月12日(周三) 凌晨0:54
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [kingToolbox/WindTerm] 拜拜苹果，那怕说的是2022年的December (Issue #429)




 
Sorry, I know it is very disappointing that the MacOS version was not released at the expected time. But believe me, although there are many unexpected things, I am still working hard to develop and test the MacOS version. Like you, I hope to release a new stable version as soon as possible.
 
At present, two important bugs have been fixed, which has greatly improved the stability, and testing is continuing. Once the testing is completed, it will be released. Please be patient, thank you.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by kingToolbox** [comment]
The MacOS version of WindTerm has been released, please download and check [WindTerm_2.3.0_Prerelease_2_MacOS_Portable](https://github.com/kingToolbox/WindTerm/releases/2.3-prerelease). If you encounter any problems, please let me know. Thank you.

**Comment by liyanggyang** [comment]
> The MacOS version of WindTerm has been released, please download and check [WindTerm_2.3.0_Prerelease_2_MacOS_Portable](https://github.com/kingToolbox/WindTerm/releases/2.3-prerelease). If you encounter any problems, please let me know. Thank you.

great

**Comment by liyanggyang** [comment]
> The MacOS version of WindTerm has been released, please download and check [WindTerm_2.3.0_Prerelease_2_MacOS_Portable](https://github.com/kingToolbox/WindTerm/releases/2.3-prerelease). If you encounter any problems, please let me know. Thank you.

M1适配有计划吗？

**Comment by kingToolbox** [comment]
Yes, it is expected that `WindTerm 2.4 (February)` or `WindTerm 2.5 (March)` will release the `Apple M1` version. We need to wait for a release cycle to see if there are still bugs that can cause crashes.

**Comment by yangxggo** [comment]
a year has passed....
